test/SILGen/objc_block_to_func_to_block.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -import-objc-header %S/Inputs/objc_block_to_func_to_block.h -emit-silgen -verify %s
```

Comment:
This is going to use the real overlays with the mock SDK, which isn't really safe.